### ISSUE TITLE
customization issue

### ISSUE THREAD

**Issue by jhindes521** [meerk40t/meerk40t#136]
having issues with the customization of axsis directions and setting home greyed out and none reponsive any ideas? 

wwin 10 opperating system installed via cmd / python

**Comment by tatarize** [comment]
That's by design. I added the GUI elements, but I neglected to actually code the stuff up. I'll push the feature up after the current version gets done. Squeaky wheels get the grease. 

I still have the issue as #86 since it's not yet finished.  It needed some gui controls to be able to be completed, but so when I was redoing the gui, I included the axis stuff. But, since I didn't have the code done, I disabled it so people wouldn't turn it on and expect it to work.

I'm currently in a feature freeze for 0.5.0 but I'll move that to the top of the list for the next version.
